Userland/Libraries/LibWeb/Streams/ReadableStreamDefaultReader.cpp

```diff
@@ -83,6 +83,7 @@ void ReadLoopReadRequest::on_chunk(JS::Value chunk)
         }
 
         m_failure_steps(exception.value());
+        return;
```

Comment:
This seems like a reasonable change but probably belongs in its own PR :)

Reply:
Sure - I just added it here because it becomes immediately obvious in the next commit, which shrinks this `if` block down to a single line

Reply:
Oh hm, fair enough!